docs/source/getting_started.rst

```diff
@@ -35,14 +35,15 @@ Installation
     MIDDLEWARE_CLASSES = [
         # ...
         'django.contrib.sessions.middleware.SessionMiddleware',
+        'wagtail.wagtailcore.middleware.SiteMiddleware',
         'wagtailtrans.middleware.TranslationMiddleware',
         'django.middleware.common.CommonMiddleware',
         # ...
     ]
 
 .. note::
 
-    Keep in mind ``wagtailtrans.middleware.TranslationMiddleware`` is a replacement for ``django.middleware.locale.LocaleMiddleware``.
+    Keep in mind ``wagtailtrans.middleware.TranslationMiddleware`` is a replacement for ``django.middleware.locale.LocaleMiddleware``. Also, it relies on ``wagtail.wagtailcore.middleware.SiteMiddleware``, which should always be mentioned first.
```

Comment:
Can we pull this apart as a separate note with a reference to the Wagtail Settings documentation? http://docs.wagtail.io/en/v1.8/getting_started/integrating_into_django.html#settings and maybe use "which should come before" - "which should always" feels somewhat dictating 😉